lib/ReactViews/CleanDropdownPanel/CleanDropdownPanel.jsx

```diff
@@ -1,4 +1,9 @@
 "use strict";
+// Ref now needs to be passed in via refForCaret as there is no longer a button
+// in CleanDropdownPanel
+
+// proptypes are in mixin
+/* eslint react/prop-types:0*/
```

Comment:
This is already set to off in `.eslint.js` on latest main.

Reply:
it is only set to off for `.ts` and `.tsx` files